spec/features/admin/index/index_spec.rb

```diff
@@ -47,4 +63,27 @@
     expect(@customers[1].last_name).to appear_before(@customers[4].last_name)
     expect(@customers[4].last_name).to appear_before(@customers[0].last_name)
   end
+
+  it "shows a list of all invoices that have items that have not yet been shipped" do
+    visit "/admin"
+
+    expect(page).to have_content("Incomplete Invoices")
+
+    expect(page).to have_content(@customer2_invoice_list[0].id)
+    expect(page).to have_content(@customer3_invoice_list[0].id)
+    expect(page).to_not have_content(@customer1_invoice_list[0].id)
+  end
+
+  it "has each invoice id which links to that invoice's admin show page" do
+    visit "/admin"
+
+    click_link("Invoice ##{@customer2_invoice_list[0].id}")
+    expect(page).to have_current_path("/admin/invoices/#{@customer2_invoice_list[0].id}")
+  end
+
+  it "Next to each invoice id I see the date that the invoice was created" do
+    visit "/admin"
+
+    expect("Friday, January 01, 2021").to appear_before("Saturday, January 01, 2022")
```

Comment:
Solid feature tests. I'm still learning about factory bot and faker but the invoice lists are starting to make sense!